config/database.yml

```diff
@@ -4,7 +4,7 @@ development: &default
   database: champaign
   pool: 5
   username: <%= ENV['PG_USERNAME'] %>
-  host: <%= ENV['PG_HOST'] %>
+  host: <%= ENV['PG_HOST'].present? ? ENV['PG_HOST'] : 'localhost' %>
```

Comment:
The more idiomatic way to express this (I think) would be [ENV.fetch](https://ruby-doc.org/core-2.2.0/ENV.html#method-c-fetch)
```ruby
ENV.fetch('PG_HOST', 'localhost')
```

Reply:
:+1: